test/check-application

```diff
@@ -1913,6 +1913,36 @@ class TestApplication(testlib.MachineCase):
         self.performContainerAction(container_name, "Resume")
         b.wait(lambda: self.getContainerAttr(container_name, "State") == "Running")
 
+    def testRenameContainerSystem(self):
+        self._testRenameContainer(True)
+
+    def testRenameContainerUser(self):
+        self._testRenameContainer(False)
+
+    def _testRenameContainer(self, auth):
+        b = self.browser
+        container_name = "rename"
+        container_name_new = "rename-new"
+        
+        self.execute(auth, f"podman container create -t --name {container_name} quay.io/libpod/busybox:latest")
+        self.login(auth)
+
+        self.filter_containers('all')
+
+        self.waitContainerRow(container_name)
+        self.toggleExpandedContainer(container_name)
+        self.performContainerAction(container_name, "Rename")
+
+        # the container name should be in the "Rename container" header
+        self.assertIn(container_name, b.text("#pf-modal-part-1"))
```

Comment:
Please use `b.wait_in_text("#pf-modal-part-1", container_name)` for this, both for consistency and also to ensure that there are no race conditions with async rendering.